sdk/identity/Azure.Identity/tests/AuthorizationCodeCredentialTests.cs

```diff
@@ -71,6 +74,26 @@ public async Task AuthenticateWithAuthCodeHonorsTenantId([Values(null, TenantIdH
             Assert.AreEqual(token2.Token, expectedToken, "Should be the expected token value");
         }
 
+        [Test]
+        [TestCaseSource(nameof(GetAllowedTenantsTestCasesWithRequiredTenantId))]
+        public async Task VerifyAllowedTenantEnforcement(AllowedTenantsTestParameters parameters)
+        {
+            Console.WriteLine(parameters.ToDebugString());
```

Comment:
Did you intend to leave this in all the tests?

Reply:
Yes. Given the input here is unique on each execution it's quite helpful in diagnosing test failures without having to re-run and debug.